main.cpp

```diff
@@ -326,14 +401,25 @@ struct Remote
     float range = 135.4f;
     int selector = 5;
     int batteries = 2;
+    Remote();
     void selectInputs();
     int adjustVolume(int volUp, int volDwn, int currVol);
     void selectMode();
 };
 
+Remote::Remote()
+{
+    brand = "";
+    modelNum = 0;
+    range = 0;
+    selector = 0;
+    batteries = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise